include/aikido/rviz/TrajectoryMarker.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef AIKIDO_RVIZ_TRAJECTORYMARKER_HPP_
+#define AIKIDO_RVIZ_TRAJECTORYMARKER_HPP_
+
+#include <dart/dynamics/Frame.hpp>
+#include <interactive_markers/interactive_marker_server.h>
+#include <visualization_msgs/InteractiveMarker.h>
+#include "aikido/trajectory/Trajectory.hpp"
+
+namespace aikido {
+namespace rviz {
+
+/// A wrapper class of RViz InteractiveMarker for visualizing AIKIDO trajectory
+/// in RViz.
+class TrajectoryMarker final
+{
+public:
+  /// Constructor.
+  /// \param[in] markerServer RViz marker server.
+  /// \param[in] frameId RViz frame ID.
+  /// \param[in] trajectory C-sapce (or joint-space) trajectory.
+  /// \param[in] skeleton Target DART meta skeleton that the C-space trajectory
```

Comment:
I found this description somewhat confusing. What do you think of "DART meta skeleton for visualizing the trajectory in task space"?